nixos/modules/services/misc/tuxclocker.nix

```
@@ -0,0 +1,72 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.tuxclocker;
+in
+{
+  options.programs.tuxclocker = {
+    enable = mkEnableOption (lib.mdDoc ''
+      TuxClocker, a hardware control and monitoring program.
```

```suggestion
      TuxClocker, a hardware control and monitoring program
```
that is already added by mkEnableOption